local/config.yaml

```diff
@@ -0,0 +1,32 @@
+controller:
```

Comment:
To satisfy the CI

```suggestion
# Unless explicitly stated otherwise all files in this repository are licensed
# under the Apache License Version 2.0.
# This product includes software developed at Datadog (https://www.datadoghq.com/).
# Copyright 2021 Datadog, Inc.
controller:
```